news/2025/2025-03-31-the-roundtable-open-at-lvl-up-expo.md

```diff
@@ -0,0 +1,99 @@
+If the pressure of playing on an unfamiliar setup wasn't enough, the stakes in this tournament will ensure your hands are drenched in sweat.
+
+![prizes image](/wiki/shared/news/2025-03-31-the-roundtable-open-at-lvl-up-expo/prizepool.jpg)
+
+## How does the tournament work?
+
+When we say this tournament is open to everyone, we mean *everyone*.
+
+The tournament is split into divisions based on skill level:
+
+- **Division 1** is for the most competitive players. [Register for division 1 here](https://www.start.gg/tournament/the-roundtable-x-lvlup-expo-division-1/details) or in-person!
+- **Division 2** is for relatively new players. [Register for division 2 here](https://www.start.gg/tournament/the-roundtable-x-lvlup-expo-division-2/details) or in-person!
+
+**You can (and should!) register for both divisions,** but if you qualify for division 1, you won't be able to participate in division 2.
+
+For the most up-to-date tournament info, it's worth monitoring [The Roundtable Open's Discord server](https://discord.com/invite/XqFDJhe922).
+
+### Tournament schedule & details
+
+Mappools for both divisions will be revealed during the event!
+
+#### Division 1
+
+| Stage | Day | Notes |
+| --: | :-- | :-- |
+| **Registration** | Before qualifiers finish | [Register here](https://www.start.gg/tournament/the-roundtable-x-lvlup-expo-division-1/details) or at the event! This division is intended for the most competitive players. |
+| **Qualifiers** | Friday | Qualifiers lobbies run hourly. After checking in on-site, you'll be assigned to a qualifiers lobby. Top 32 players from qualifiers will move onto the bracket stage. |
+| **Round of 32** | Saturday | Best of 7, 1 ban |
+| **Round of 16** | Saturday | Best of 7, 1 ban |
+| **Quarterfinals** | Saturday | Best of 9, 1 ban |
+| **Semifinals** | Saturday | Best of 9, 1 ban |
+| **Finals** | Saturday | Best of 9, 2 bans |
+| **Grand Finals** | Saturday | Best of 11, 2 bans |
+
+#### Division 2
+
+| Stage | Day | Notes |
+| --: | :-- | :-- |
+| **Registration** | Before qualifiers finish | [Register here](https://www.start.gg/tournament/the-roundtable-x-lvlup-expo-division-2/details) or at the event! This division is intended for semi-casual players who want to experience osu! in a competitive setting. |
+| **Qualifiers** | Saturday | Qualifiers lobbies run hourly. After checking in on-site, you'll be assigned to a qualifiers lobby. Top 16 players from qualifiers will move onto the bracket stage. |
+| **Round of 16** | Sunday | Best of 7, 1 ban |
+| **Quarterfinals** | Sunday | Best of 9, 1 ban |
+| **Semifinals** | Sunday | Best of 9, 1 ban |
+| **Finals** | Sunday | Best of 9, 2 bans |
+| **Grand Finals** | Sunday | Best of 11, 2 bans |
+
+## Can I help out?
+
+If your dream is to wrangle osu! players to their designated PCs, you can [apply to be a volunteer for the tournament here!](https://x.com/TheRoundtableLA/status/1900661907608268990) If accepted, you'll receive a 3-day volunteer badge at no cost.
```

Comment:
would be convenient to not have to click twice and wait for page loads or be interrupted by twitter login walls

```suggestion
If your dream is to wrangle osu! players to their designated PCs, you can [apply to be a volunteer for the tournament here!](https://docs.google.com/forms/d/e/1FAIpQLSex7f1YZfbUQbmR8gdb7_7dy-rNFk0Tz7Y6o7s04V7h7S0ooQ/viewform) If accepted, you'll receive a 3-day volunteer badge at no cost.
```